R/path.R

```diff
@@ -131,11 +132,10 @@ path_abs <- function(path, start = ".") {
 }
 
 
-#' @describeIn path_math collapses redundant separators and
-#' up-level references, so `A//B`, `A/B`, `A/./B` and `A/foo/../B` all become
-#' `A/B`. If one of the paths is a symbolic link, this may change the meaning
-#' of the path, in this case one should use [path_real()] prior to calling
-#' [path_norm()].
+#' @describeIn path_math collapses redundant separators and up-level references,
+#'   so `A//B`, `A/B`, `A/./B` and `A/foo/../B` all become `A/B`. If one of the
+#'   paths is a symbolic link, this may change the meaning of the path, in this
+#'   case one should use `path_real()` prior to calling `path_norm()`.
 #' @export
```

Comment:
Re: redundant separators, is that just the background ubiquitous application of `path_tidy()`? Or is this alluding to something else?

Re: advice to call `path_real()` and then `path_norm()`. In my hands, `path_real()` seems to basically doing everything `path_norm()` does, and then some. There would be no point in calling `path_norm()` after `path_real()` because I don't think up-references survive.

``` r
library(fs)

dir_create("a")
file_create("a/b")
link_create(path_abs("a"), "c")

# Realize the path
path_real("c/b")
#> /private/var/folders/vr/gzrbtprx6ybg85y5pvwm1ct40000gn/T/RtmpWYfmhr/a/b

# I don't think you can realize a path without also normalizing it (??)
path_real("c/./b/..")
#> /private/var/folders/vr/gzrbtprx6ybg85y5pvwm1ct40000gn/T/RtmpWYfmhr/a
```

Created on 2018-06-15 by the [reprex package](http://reprex.tidyverse.org) (v0.2.0).

Reply:
Yes I think we can remove the 'in this case...' fragment of the sentence